cmd/src-fingerprint/main.go

```diff
@@ -210,6 +185,12 @@ func main() {
 						Value: DefaultTimeout,
 						Usage: "Maximum time to process each object (0 for unlimited, min. 1s).",
 					},
+					&cli.UintFlag{
+						Name:  "pool",
+						Value: 1,
+						Usage: "Size of the pool of worker to process objects simultaneously." +
+							"More workers means more memory and it compounds with the number of cloners.",
```

Comment:
Do you think we should add some kind of limitations on pool and cloners given the machine capabilities ?